tests/cmsis_rtos_v1/src/thread_apis.c

```diff
@@ -123,4 +136,11 @@ void test_thread_apis(void)
 	do {
 		osDelay(100);
 	} while (thread_yield_check != 2);
+
+	osThreadTerminate(id1);
+	osThreadTerminate(id2);
+
+	/* Try to terminate inactive thread and assert failure */
+	status = osThreadTerminate(id1);
+	zassert_false(status == osOK, "Inactive thread cannot be terminated");
```

Comment:
Move this code to the end of "test_thread_prio()". We are already having a call to osThreadTerminate() there. So, the extra calls can be removed.
Check for osErrorResource. The statement can be something like
zassert_true(status == osErrorResource, "Something's wrong with osThreadTerminate");